backend/contracts/index.ts

```diff
@@ -1 +1,3 @@
-export * from './user.interface';
\ No newline at end of file
+export * from './ICoffeeProfile';
+export * from './IRequest';
+export * from './IUser';
```

Comment:
I like this naming convention instead of .interface!

Reply:
`.interface` is an Angular convention I believe. It's probably in there docs or something lol